.github/workflows/rust.yml

```diff
@@ -1,49 +1,49 @@
-name: Rust
+name: Rust CI
 
 on:
   push:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  schedule:
+    - cron: "0 0 * * *"
 
 env:
+  RUST_BACKTRACE: 1
+  CARGO_TERM_VERBOSE: true
   CARGO_TERM_COLOR: always
-  LIB: 'c:\winpcap-dp\WpdPack\Lib\x64'
 
 jobs:
-  build:
-
-    runs-on: windows-latest
+  cargo-test:
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+        profile: ['', --release]
+        toolchain: [stable]
+        features: ['']
```

Comment:
You disabled features testing (`tokio`, `full`).
If you enable `tokio` will CI fail? (Previously succeeded)

Reply:
Yes `--features tokio` fails. #120